.github/CODEOWNERS

```
@@ -1,3 +1,3 @@
 * @delapuente @y4izus @techtolentino @korgan00 @eddybrando
```

`abdonrd` was still missing:

```suggestion
* @delapuente @y4izus @techtolentino @korgan00 @eddybrando @abdonrd
```